sml_small/editing/thousand_pounds/thousand_pounds.py

```diff
@@ -1,15 +1,37 @@
 from dataclasses import dataclass
+from enum import Enum
 from typing import List, Optional
 
+from sml_small.utils.common_utils import validate_number
+from sml_small.utils.error_utils import get_boundary_error, get_mandatory_param_error, get_one_of_params_mandatory_error
+
+
```

Comment:
In Totals and Components we have a comment to separate out Constant, Enum, Class, Exception definitions.  Could you align the code with that style please.

# ---- Constant Definitions ----
# ---- Enum Definitions ----
# ---- Class Definitions ----
# ---- Custom Exceptions ----
# ---- Method Definitions ----